src/utils/format-date.js

```diff
@@ -98,7 +101,17 @@ export function formatDate(d) {
     formattedUtcShortDate: `${
       shortMonths[date.getUTCMonth()]
     } ${date.getUTCDate()}, ${date.getUTCFullYear()}`,
-    clockTimeUtc: `${utcTimeTwelve.join(":")} ${utcAMPM} - UTC-0`
+    clockTimeUtc: `${utcTimeTwelve.join(":")} ${utcAMPM} - UTC`,
+    formattedTimezone: `${
```

Comment:
Also, lets use 24 hour time, remove the AM/PM stuff. 

The design actually uses 24 hour time, didn't use it on master for existing consistency. 
![image](https://user-images.githubusercontent.com/3970376/56311158-f5a44580-6112-11e9-86fa-c22518544eb9.png)